documentation/book/assembly-hello-world.adoc

```diff
@@ -0,0 +1,46 @@
+// This assembly is included in the following assemblies:
+//
+// getting-started.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+:parent-context: {context}
+
+[id='hello-world-{context}']
+
+== Hello world example
+
+You can try this Hello world example to ensure everything is working fine for you.
+This example consists from producer and consumer which are communicating via Kafka cluster.
+
+. Setup environment
++
+You can download latest version of {ProductName} project from {ProductName} GitHub repository https://github.com/strimzi/strimzi-kafka-operator.
+Unzip archive by command `unzip strimzi-kafka-operator-master` or in your system GUI.
+
+. Deploy example cluster
++
+At first you have to install all resources. You can do that by running this command from cloned (unzipped) {ProductName} folder.
++
+`oc apply -f examples/install/cluster-operator`
++
+Now you are ready to deploy example cluster by executing following command.
++
+`oc apply -f examples/kafka/kafka-ephemeral.yaml`
+
+. Run producer and consumer
++
+Now the pods are starting. Their status can obtained by command `oc get pods` or you can open web console at address `127.0.0.1:8443` and watch the status here.
+When all pods are successfully started you can start producing messages by deploying producer.
++
+`oc run kafka-producer -ti --image=strimzi/kafka-base:latest --restart=Never \-- bin/kafka-console-producer.sh --broker-list my-cluster-kafka-bootstrap:9092 --topic test`
```

Comment:
The base image is different upstream and downstream. You have to add variable to `attributes.adoc` and use a placeholder here. Also the `my-cluster` adn the topic should be marked as somethign the user should replace with his own cluster name.